index.html

```diff
@@ -0,0 +1,183 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css">
+    <script
+      src="https://kit.fontawesome.com/5d13e94c8b.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <header class="header">
+        <div class="header-div">
+            <div>
+            <ul class="icons-ul">                
+                <li><img class="head-icon" src="./images/faceb.png" alt="facebookicon"></li>
+                <li><img class="head-icon" src="./images/twtr.png" alt="twitter-icon"></li>
+                <li class="link">Other language</li>
+                <li class="link">my page</li>
+                <li class="link">Logout</li>
+            </ul>
+            </div>
+            
+            <div class="bars">
+        </div>
+                 <nav>
+                    <div class="span-"><img  src="./images/download.jpg" alt="#" class="inner-backg"><p class="logo">LIVING FAITH CHURCH</p> </div>
+
+                    <ul class="nav-ul">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="index.html">Home</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li><h3 class="winners">Winners Chapel</h3></li>
+                    </ul>
+                </nav>
+            </div>
+    </header>
+    
+    <section>
+        <article class="intro">
+            <div class="menu-nav">
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+          </div>
+
+           
+<div class="mobileMenu"> 
+  
+<div class="m1"><h2 class="m2">Winners Chapel</h2></div>
+
+<span class="closeMenu">&times;</span>
+  <ul class="container">
+        <li><a href="about.html">About</a></li>
+        <li><a href="index.html">Home</a></li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>News</li>
+ </ul>
+
+</div>
+        </article>
+
+    </section>
+
+    <main>
+        
+<div class="pg1">
+    <h2 class="theme">
+        "Youth For Christ"
+     </h2>
+     <h1 class="ch-name">Winners Chapel Youth Conference 2022.</h1>
+    <div class="moment">
+         <p class="prg">in His presnce there is fullness of joy, experience God in another demention come youth 2022 conference</p>
+    </div>
+     <div class="info">
+        <h6 class="date">22.05.2022(THUR)~23(FRI)</h6>
+        <p class="venue">@ Church Auditorium, Lagos.</p>
+    </div>
+ </div>
+
+ <section class="prog-card">
+    <h5 class="h5">Main program</h5>
+    <hr class="hr2">
+    <div class="card-section">
+        <div class="card">
+            <div class="card-info">
+            <ul class="card-content">
+                    <li><img class="img-w" src="./images/lecture.png" alt="#"></li>
```

Comment:
- [ ] Please always make sure to give a proper `alt` value. This value helps to improve accessibility and is considered a good practice. Please add an appropriate `alt` value for all the `img` tags